__tests__/codeowners.spec.ts

```diff
@@ -1,10 +1,9 @@
 import { loadCodeOwnerFiles } from '../src/utils/codeowners';
-import { readContent } from '../src/utils/readContent';
-import { mocked } from 'ts-jest/utils';
+import * as readContent from '../src/utils/readContent';
 
 jest.mock('../src/utils/readContent');
 
-const readContentMock = mocked(readContent);
+const readContentMock = readContent.readContent as jest.Mock<ReturnType<typeof readContent.readContent>>;
```

Comment:
You can also use `jest.mocked`, introduced in `jest` version `27.4.4` (https://github.com/facebook/jest/releases/tag/v27.4.4). It does the same thing as the `ts-jest` function.